tests/dummy/app/templates/docs/atoms/au-form-radio.md

```diff
@@ -9,10 +9,10 @@
     <div class="au-c-form">
       <ul>
         <li>
-          <AuControlRadio @label="Ja" @name="ja-nee" @value="ja" @identifier="ja" />
+          <AuControlRadio @label="Ja" @name="ja-nee"  @identifier="ja" value="ja" />
```

Comment:
moving from `@value` to `value` is a breaking change

Reply:
Is it possible to remove the @ on every instance? it will still work the same when this is done, I prioritized the "@checked" over "@value"  since that one is more likely to need two-way binding rather the "value" (linting error doesnt like you to use both @checked & @value in an Input component)